### DISCUSSION TITLE
Units of Solution data

### DISCUSSION THREAD

**Discussion by nikmarlo** [ansys/pyaedt#5143] [Q&A] [answered]
Hi,
I am trying to obtain the solution data of an inductance matrix in Maxwell 2D. My code to extract the individual values is as follows:
```
quants = maxwell_2d.post.available_report_quantities(context="coupling", quantities_category='L')

L_list = []
for komp in quants:
    coupler = maxwell_2d.post.get_solution_data('coupling.'+komp)
    L_list.extend(coupler.data_real())

L_mat = np.array(L_list).reshape(3, 3)
```
While this works well, I can't get the information about unit of the results. Is there any way to load the inductance unit information via code, or can I assume that the unit will always be [nH] by default (which is the unit in my case)?

Thanks a lot!

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @nikmarlo , you can get the units in 2 different ways.

From the data directly:

coupler.units_data

I encourage you to debug your code, and do:

dir(coupler)

You will get all available properties.

And also you can get the AEDT internal units:

maxwell_2d.odesktop.GetDefaultUnit("Inductance")
